packages/riipen-ui/src/components/Table.jsx

```diff
@@ -0,0 +1,241 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import ThemeContext from "../styles/ThemeContext";
+
+import TableColumn from "./TableColumn";
+import TableHeader from "./TableHeader";
+import TableContainer from "./TableContainer";
+import TableRow from "./TableRow";
+import TableBody from "./TableBody";
+import TableCell from "./TableCell";
+import TableHeaderCell from "./TableHeaderCell";
+
+class Table extends React.Component {
+  static propTypes = {
+    /**
+     * Displays the label instead of the header
+     */
+    allowSearch: PropTypes.bool,
+
+    /**
+     * The children columns to render
+     * Must be of type TableColumn
+     */
+    children: PropTypes.oneOfType([
+      PropTypes.arrayOf(
+        PropTypes.shape({ type: PropTypes.oneOf([TableColumn]) })
+      ),
+      PropTypes.shape({ type: PropTypes.oneOf([TableColumn]) })
+    ]),
+
+    /**
+     * Classes to apply to the root element
+     */
+    classes: PropTypes.arrayOf(PropTypes.string),
+
+    /**
+     * The entities to render in the table
+     */
+    entities: PropTypes.array,
```

Comment:
What are table "entities"? Can we make this documentation more clear. Are these rows?

Reply:
we could do something like rowData or rowEnitities, this is the array of data that gets looped through to generate the table. Each value in the array is used in the call back for the cell render function

Reply:
I like `rowData`

Reply:
I think even just "data", I was looking at other table implementations and react-tables takes in "data" and "columns" to construct everything you need.